cmd/container-suseconnect/main.go

```diff
@@ -89,6 +92,9 @@ func main() {
 		appAction = runZypperURLResolver
 	case "z", "zypp", "zypper":
 		appAction = runZypperPlugin
+	default:
+		flag.Usage()
+		os.Exit(1)
```

Comment:
until 2.5.0, container-suseconnect never exited with 1 (at least I can't find one case), even if you call the wrong arguments, it will default to `runListProducts`.

I don't think this is correct as it can still break things in some unknown scenarios.

Reply:
well, I think it is important to highlight that an arg was given but not parsed. we can't just put that under the carpet and pretend nothing has happened.

Reply:
(I'm fine with calling it `v2.6.0`if you think that is helping with this conversation)

Reply:
I'm trying not to break any existing use cases that we might not be aware of.

We could do a v3 breaking change if we test things properly and ensure all our use cases still work, but currently this project lacks tests for the CLI part.

Reply:
We could write erroneous arguments to the log file, at least, it would be traced somewhere, and we could make a warning that in v3 or current + N releases (for some buffer) will be a breaking behavior, at least people are warned about it.

Reply:
@alexandrevicenzi which breaking behavior are you talking about ? we are not breaking the zypper plugin case here.

Reply:
With the last changes, I think we don't have any breaking changes to the zypper plugin.